src/pages/DashboardDoctor/components/AddCheckup.tsx

```diff
@@ -0,0 +1,86 @@
+import { useEffect, useState } from "react";
+import { useForm } from "react-hook-form";
+import { useHistory } from "react-router";
+import { addCheckup } from "../../../api/checkups";
+import { getTemplatesByHospital } from "../../../api/templates";
+import { INewCheckup } from "../../../models/ICheckup";
+import { ITemplate } from "../../../models/ITemplate";
+
+const AddCheckup = () => {
+  const [template, setTemplate] = useState<ITemplate>();
+  const history = useHistory();
+  const { register, handleSubmit } = useForm();
+
+  const onSubmit = handleSubmit((data) => {
+    const doctorId = Number(localStorage.getItem("doctorId") as string);
+    const { patient, ...responses } = data;
+
+    const newCheckup: INewCheckup = {
+      doctor_id: doctorId,
+      template_id: 1,
```

Comment:
Shouldn't the `template_id` come from a variable rather than a hardcoded value ?